spec/visual/samples/php

```diff
@@ -291,11 +374,11 @@ class Zip extends Archive {
     // Return content?
     if(!$filename)
       return $data;
-      
+  
     // Write to file
     return file_put_contents($filename, $data);
   }
-  
+
  /**
```

Comment:
I realise that this isn't actually from your code but this looks like it's rendering incorrectly in the visual test app. It's supposed to be a multiline comment, right?

Reply:
These are just trailing whitespaces that I've accidentally removed with my editor, seem to be rendered just fine though.

Reply:
Sorry, my comment wasn't very clear. In the visual sample there is this text:

```
 /**
 *  Zip class file
 *
 *  @package     fnord.bb
 *  @subpackage  archive
 */
```

It's being rendered as a string but it should be a comment, right?

Just to be clear, your code hasn't caused this error (if it is an error); it's there right now. I was just taking advantage of asking you since you seemed more knowledgeable about PHP than me :D

Reply:
ah, I see what you mean, I just saw it was marked up in a colour and assumed it was the right one, yes you are right.

Reply:
Fixed this with ca1d28c